### DISCUSSION TITLE
Welcome to Doodle Discussions!

### DISCUSSION THREAD

**Discussion by pusolito** [nacular/doodle#6] [General]
## 👋 Welcome!
  These Discussions are a place to connect with other members of our community. Please use them to:
  * Ask questions you’re wondering about
  * Share ideas
  * Engage with other community members
  * Welcome others

  To get started, comment below with an introduction of yourself and how you're using Doodle.

**Comment by WiseNN** [comment]
Hey Guys, just dropped by to offer some encouragement. I have just discovered this repo. I really love this project. Keep going!
